src/web/client/WebExtensionContext.ts

```diff
@@ -56,6 +58,8 @@ export interface IWebExtensionContext {
     defaultFileUri: vscode.Uri; // This will default to home page or current page in multifile scenario
     showMultifileInVSCode: boolean;
     extensionActivationTime: number;
+    treeWebView: TreeWebViewProvider;
```

Comment:
Do we really need this to be added in context interface?

Reply:
yes in my opinion as in further ms graph service implementation, we need to call this provider there also